### PR TITLE
✨ Add NavBar + Link

### LINKED ISSUE #3: Burger menu
#### Description

The Burger menu should navigate to the setting page

#### Screenshots

image

#### Tasks

Include specific tasks in the order they need to be done in. Include links to specific lines of code where the task should happen at.

- [x] Create the burger button 
- [x] Link to Setting page

### PR DESCRIPTION
**Pull request by JGieseWeb**
hotfix the preview.js

### CODE REVIEW

**Review by Dev-dfm — APPROVED** [review]
lgtm 👍

**Review by Niklas-Schaefers — APPROVED** [review]
lgtm 🚀

File: src/app/components/Icons/BurgerMenuIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+export default function BurgerMenuIcon(): JSX.Element {
+  return (
+    <svg
+      id="Menu"
```
Comment (lmachens) [inline src/app/components/Icons/BurgerMenuIcon.tsx]:
```suggestion
```
You don't need an id here :)

File: src/app/components/NavBar/NavBar.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import NavBar from "./NavBar";
+
+export default {
+  title: "Component/NavBar",
+  component: NavBar,
+};
+export const NavBarContainer = (): JSX.Element => <NavBar />;
```
Comment (lmachens) [inline src/app/components/NavBar/NavBar.stories.tsx]: It's not a `Container`. Consider choosing a different name:

```suggestion
export const Default = (): JSX.Element => <NavBar />;
```

```suggestion
export const Default = (): JSX.Element => <NavBar />;
```

**Review by lmachens — COMMENTED** [review]
Good job, I like the NavBarLink component

File: src/app/components/NavBar/NavBar.module.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.navBar {
+  margin: 0;
+}
```
Comment (lmachens) [inline src/app/components/NavBar/NavBar.module.css:3]: I don't think this is required. The default should be `0`

**Review by lmachens — APPROVED** [review]
Good job

**Pull request merged**